miner/scroll_worker.go

```diff
@@ -751,57 +837,69 @@ func copyReceipts(receipts []*types.Receipt) []*types.Receipt {
 	return result
 }
 
-func (w *worker) onTxFailingInPipeline(txIndex int, tx *types.Transaction, err error) bool {
+func (w *worker) onTxFailing(txIndex int, tx *types.Transaction, err error) bool {
```

Comment:
we can add a comment about what the returned value stands for

Reply:
The return value has no use. removed https://github.com/scroll-tech/go-ethereum/pull/982/commits/024a49c01c2d26efc279c8c4abbea7baff544640.